cfgov/unprocessed/css/misc.less

```diff
@@ -271,6 +271,16 @@
     }
 }
 
+.archived-page main {
+  img,
+  .m-hero,
+  .m-info-unit_image,
+  .embed-container,
+  iframe {
+    filter: grayscale();
+  }
+}
+
```

Comment:
Trivial: Our standard for CSS/Less is 4-space indentation.

Reply:
Updated in https://github.com/cfpb/consumerfinance.gov/pull/6038/commits/6aed924cf92ba897cce6715791d033a41c55bc7f.